easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb

```diff
@@ -0,0 +1,23 @@
+name = 'MATLAB'
+version = '2018b'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# be sure to copy both DVD content to the SAME directory, 
+# including the hidden files, especially .dvd1 and .dvd2 
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Java', '1.8')]
+
+license_server = 'my-license-server'
+license_server_port = '1234'
+key = '00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000'
```

Comment:
@vanzod These should be commented out, so the MATLAB easyblock can pick up on `$EB_MATLAB_KEY` & co...

Reply:
Yes, and to cover the cases where server-specific licenses are being used.

Reply:
Done. I also added a short comment for clarity